broker/src/main/java/io/zeebe/broker/transport/commandapi/CommandApiRequestHandler.java

```diff
@@ -81,6 +82,17 @@ private void handleExecuteCommandRequest(
       final DirectBuffer buffer,
       final int messageOffset,
       final int messageLength) {
+
+    if (!isDiskSpaceAvailable) {
+      errorResponseWriter
+          .resourceExhausted(
+              String.format(
+                  "Cannot accept requests for partition %d. Broker is out of disk space",
```

Comment:
Can we write the current disk space and the threshold as context info?

Reply:
Is it useful for the clients? We log it in `DiskSpaceUsageMonitor`.

Reply:
It's useful for us that's for sure, hard to say yet if it's useful for normal users. Do you think adding it would confuse them? What are the reasons for omitting them?

Reply:
CommandApi does not now how much disk space is available or required. It only knows that there is not enough. If we want to provide this information to the users, the listener interface should be changed. Since user's can't do anything with that information, I don't think it is necessary. We log both required and available disk space in DiskSpaceUsageMonitor, which can be used by an operator.